src/common/quantile.cc

```diff
@@ -445,29 +449,27 @@ void SketchContainerImpl<WQSketch>::MakeCuts(Context const *ctx, MetaInfo const
       max_cat = std::max(max_cat, AddCategories(categories_.at(fid), p_cuts));
```

Comment:
Based on my understanding, categorical features are not yet supported right?

Reply:
right, need to find a proper testing data with categorical features